charts/signoz/README.md

```diff
@@ -65,210 +65,217 @@ kubectl delete namespace platform
+| `queryService.initContainers.init.image.repository`         | Query Service initContainer image name                                                                     | `busybox`                      |
+| `queryService.initContainers.init.image.tag`                | Query Service initContainer image tag                                                                      | `1.35`                         |
+| `queryService.initContainers.init.image.pullPolicy`         | Query Service initContainer pull policy                                                                    | `IfNotPresent`                 |
+| `queryService.initContainers.init.command`                  | Query Service initContainer command line to execute                                                        | See `values.yaml` for defaults |
+| `queryService.initContainers.init.resources`                | Resources requests and limits                                                                              | See `values.yaml` for defaults |
+| `queryService.additionalEnvs`                               | Additional environment variables for query-service container                                               | `[]`                           |
+| `queryService.configVars`                                   | Query Service configurations                                                                               | See `values.yaml` for defaults |
+| queryService.smtpVars.enabled                               | Enable SMTP for user invitation. It will set `SMTP_ENABLED` when enabled.                                  | `false`                        |
+| queryService.smtpVars.existingSecret.fromKey                | Name of key in secret to get value for `SMTP_FROM`. If empty, will not set the env variable.               | `""`                           | 
+| queryService.smtpVars.existingSecret.hostKey                | Name of key in secret to get value for `SMTP_HOST`. If empty will not set the env variable.                | `""`                           | 
+| queryService.smtpVars.existingSecret.name                   | Name fo the existing k8s secret to pick the values. Needed when one of the key for existing secret is set. | `""`                           | 
+| queryService.smtpVars.existingSecret.passwordKey            | Name of key in secret to get value for `SMTP_PASSWORD`. If empty will not set the env variable.            | `""`                           | 
+| queryService.smtpVars.existingSecret.portKey                | Name of key in secret to get value for `SMTP_PORT`. If empty will not set the env variable.                | `""`                           | 
+| queryService.smtpVars.existingSecret.usernameKey            | Name of key in secret to get value for `SMTP_USERNAME`. If empty will not set the env variable.            | `""`                           | 
```

Comment:
```suggestion
| `queryService.smtpVars.enabled`                               | Enable SMTP for user invitation. It will set `SMTP_ENABLED` when enabled.                                  | `false`                        |
| `queryService.smtpVars.existingSecret.fromKey`                | Name of key in secret to get value for `SMTP_FROM`. If empty, will not set the env variable.               | `""`                           | 
| `queryService.smtpVars.existingSecret.hostKey`                | Name of key in secret to get value for `SMTP_HOST`. If empty will not set the env variable.                | `""`                           | 
| `queryService.smtpVars.existingSecret.name`                   | Name fo the existing k8s secret to pick the values. Needed when one of the key for existing secret is set. | `""`                           | 
| `queryService.smtpVars.existingSecret.passwordKey`            | Name of key in secret to get value for `SMTP_PASSWORD`. If empty will not set the env variable.            | `""`                           | 
| `queryService.smtpVars.existingSecret.portKey`                | Name of key in secret to get value for `SMTP_PORT`. If empty will not set the env variable.                | `""`                           | 
| `queryService.smtpVars.existingSecret.usernameKey`            | Name of key in secret to get value for `SMTP_USERNAME`. If empty will not set the env variable.            | `""`                           | 
```

Reply:
@umarhussain15 one small comment for documentation